doc/rtd/development/packaging.rst

```diff
@@ -0,0 +1,93 @@
+.. _downstream_packaging:
+
+Downstream packaging
+********************
+
+This page is intended to support operating system packagers of ``cloud-init``
+and is not intended for other audiences to generate their own custom cloud-init
+packages. One should first look to official downstream cloud images containing
+pre-built packages of ``cloud-init`` for your preferred operating system.
+
+``Cloud-init`` is distributed as a system package and pre-installed in cloud
+images. Correct behavior depends on the tight integration of system services
+performing system-wide configuration during
+:ref:`the early boot stages<boot_stages>`. Because of this, cloud-init is not
+published to PyPI as it is not intended to be consumed as a pure-python
+package or run from virtual environments or python paths that are not
+system-wide.
+
+``Cloud-init`` upstream uses meson_ as a build backend to package dependent
+data files, system-initialization scripts and global configuration files and
+deliver python modules to global ``site-packages`` paths.
```

Comment:
This information isn't necessary. Every build system does some variation of this.

Reply:
ok, dropped.